templates/serverpod_templates/projectname_server_upgrade/docker-compose.production.yaml

```diff
@@ -0,0 +1,120 @@
+services:
+  traefik:
+    restart: on-failure
+    image: traefik:v2.3
+    command:
+      - "--api.insecure=true"
+      - "--providers.docker=true"
+      - "--entrypoints.web.address=:80"
+      - "--entrypoints.websecure.address=:443"
+      - "--certificatesresolvers.myresolver.acme.tlschallenge=true"
+      # TODO Change this to your email address, Let's Encrypt will use this to notify you about certificate expiration
+      - "--certificatesresolvers.myresolver.acme.email=damian_and_paul@examplepod.com"
+      - "--certificatesresolvers.myresolver.acme.storage=/letsencrypt/acme.json"
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock"
+      - "./letsencrypt:/letsencrypt"
+    depends_on:
+      - postgres
+      - serverpod
+    networks:
+      - serverpod-network
+
+  postgres:
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "postgres"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    restart: on-failure
+    image: postgres:14.1
```

Comment:
Suggestion: The current development template is using postgres 16, e.g. bump it!